chalice/cli/__init__.py

```diff
@@ -79,11 +80,14 @@ def cli(ctx, project_dir, debug=False):
 @click.option('--port', default=8000, type=click.INT)
 @click.option('--stage', default=DEFAULT_STAGE_NAME,
               help='Name of the Chalice stage for the local server to use.')
+@click.option('--autoreload', is_flag=True)
 @click.pass_context
-def local(ctx, host='127.0.0.1', port=8000, stage=DEFAULT_STAGE_NAME):
-    # type: (click.Context, str, int, str) -> None
+def local(ctx, host='127.0.0.1', port=8000, stage=DEFAULT_STAGE_NAME,
+          autoreload=False):
+    # type: (click.Context, str, int, str, bool) -> None
     factory = ctx.obj['factory']  # type: CLIFactory
-    run_local_server(factory, host, port, stage, os.environ)
+    with Reloader(autoreload):
```

Comment:
Seems a little bit strange to me to pass `False` to class called Reloader. Could we refactor this to just not use the Reloader context manager if its disabled?

Reply:
You are right, I'll fix it.